src/main/java/com/facebook/openwifirrm/ucentral/models/State.java

```diff
@@ -112,8 +112,21 @@ public class Memory {
 
 	public Unit unit;
 
+	public class Radio {
+		public long active_ms;
+		public long busy_ms;
+		public int channel;
```

Comment:
I'm not 100% sure these fields were always non-null... I think it's probably safe to assume `channel` 0 = uninitialized, less sure about `tx_power` 0 since that's a valid value(?). Let's be careful here and do some more testing... for example if you can reset an AP to default *configuration* (can you do this by deleting "channel" and "tx-power" keys?) then what do the stats (this object) return?

Reply:
will try to see how I can test this more

Reply:
Tried deleting the configuration, if tx power and channel are not available then `radios` is just not populated at all